client/src/components/Event.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+function Event ({props}) {
+    let ratingStyle = {
+        'color': '#00a680',
+        'border': '#00a680 1px solid',
+        'padding': '2px',
+        'borderRadius': '25px',
+        'fontSize': '.5em'
+    }
+
+    let ratingStyleWhite = {
+        'color': 'white',
+        'border': '#00a680 1px solid',
+        'padding': '2px',
+        'borderRadius': '25px',
+        'fontSize': '.5em'
+    }
+
+    let ratings = [];
+    for (let index = 0; index < 5; index++) {
+        if (index < Math.round(props.rating)) { // TODO: Show half dots for ratings such as 4.5
+            ratings.push(0);
+        } else {
+            ratings.push(1);
+        }
+    }
+    
```

Comment:
Curious, if you are passing a ratings param, why do you need to create a ranged array?

Reply:
@WormJim The array tells `<Event/>` where to render dots and empty circles. For a rating of 4, the array would be [0,0,0,0,1]. 

`<Event/>` renders the dots and circles by invoking the `map` method on the array.

I prefer using only the `for-loop` and not `map`, since they serve the same purpose here. However, I get errors if the `for-loop` is inside my React component. So that's why I create the ranged array before rendering `<Event/>`.